build/teamcity-test-sql.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+tc_prepare
+
+export TMPDIR=$PWD/artifacts/test
+mkdir -p "$TMPDIR"
+
+tc_start_block "Compile C dependencies"
+# Buffer noisy output and only print it on failure.
+run build/builder.sh make -Otarget c-deps &> artifacts/c-build.log || (cat artifacts/c-build.log && false)
+rm artifacts/c-build.log
+tc_end_block "Compile C dependencies"
+
+tc_start_block "Run Go tests (./pkg/sql/logictest only)"
+export PKG=$(go list ./pkg/... | grep "pkg/sql/logictest" | sed 's/github.com\/cockroachdb\/cockroach/./g' | paste -sd " " -)
```

Comment:
tip: `sed 's,github.com/cockroachdb/cockroach,.,g'` so you don't need to escape `/`

Also are you sure that `paste` is needed? Does `make test` reject a newline-separated PKG?